docs/overview/diagrams.md

```diff
@@ -121,6 +121,47 @@ classDiagram
 
+# AssociativeEither
+
+[![AssociativeCompose hierarchy][AssociativeEither-image]][AssociativeEither-link]
+
+```mermaid
+classDiagram
+  AssociativeEither~F<_>~ <|-- CommutativeEither~F<_>~
+  AssociativeEither~F<_>~ <|-- IdentityEither~F<_>~
+  class AssociativeEither~F<_>~{
+    Either[L, +*]
+    Exit[E, +*]
+    Fiber[E, +*]
+    Future[+*]
+    Schedule[R, E, +*]
+    Try[+*]
+    ZIO[R, E, +*]
+    ZLayer[R, E, +*]
+    ZManaged[R, E, +*]
+    ZStream[R, E, +*]
+    
+    () either[A, B](() => F[A], () => F[B]): F[Either[A, B]]
+  }
+  class CommutativeEither~F<_>~{
+    Future
```

Comment:
`Future[+*]`? But please check that Future has a covariant type parameter, please :smile:

Reply:
checked Future has a covariant type parameter, also I removed Future[+*] from AssociativeEither because it has been implemented on CommutativeEither